File: sdks/python/examples/verified_computing_node.py
import requests

from alith.lazai import ProofRequest

proofRequest = ProofRequest(
    job_id=1,
    file_id=1,
    file_url="1",
    encryption_key="2",
    encryption_seed="3",
    proof_url=None,
    nonce=None,
)

response = requests.post("http://127.0.0.1:80/proof", json=proofRequest.model_dump())

if response.status_code == 200:
    try:
        print("Proof request sent successfully")
        print("Response data:", response.json())
    except requests.exceptions.JSONDecodeError:
        print("Server returned non-JSON content:", response.text)
else:
    print(f"Failed with status code {response.status_code}")
    print("Response content:", response.text)

File: sdks/node/examples/verified_computing_node.ts
import axios, { type AxiosResponse } from "axios";
const proofRequest = {
  job_id: 16,
  file_id: 17,
  file_url: "1",
  encryption_key: "2",
  encryption_seed: "3",
  nonce: null,
  proof_url: null,
};
const response: AxiosResponse = await axios.post(
  "http://127.0.0.1:80/proof",
  proofRequest,
  {
    headers: { "Content-Type": "application/json" },
  }
);
if (response.status === 200) {
  console.log("Proof request sent successfully");
} else {
  console.log("Failed to send proof request:", response.data);
}
